### PR TITLE
Judicial system: User Control

### PR DESCRIPTION
**Pull request by gudjong**
# User Control

https://app.asana.com/0/1199153462262248/1199582364552602

## What

☝️ 

## Why

Users are currently managed through database seeds, which is not ideal.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/api/src/app/modules/user/dto/createUser.input.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  @Allow()
+  @Field()
+  readonly mobileNumber!: string
+
+  @Allow()
+  @Field()
+  readonly email!: string
+
+  @Allow()
+  @Field(() => String)
```
Comment (oddsson) [inline apps/judicial-system/api/src/app/modules/user/dto/createUser.input.ts:30]: Is this correct? Why is this needed here but not in the other fields?

**Review by oddsson — APPROVED** [review]

File: apps/judicial-system/api/src/app/modules/user/dto/createUser.input.ts (hunk 1 above)
Comment (gudjong) [reply apps/judicial-system/api/src/app/modules/user/dto/createUser.input.ts:30]: The reason is the enum type UserRole. The GraphQl schema generator cannot resolve it without help.
